src/renderer/page/file/view.jsx

```diff
@@ -17,6 +17,7 @@ class FilePage extends React.PureComponent {
   componentDidMount() {
     this.fetchFileInfo(this.props);
     this.fetchCostInfo(this.props);
+    this.checkSubscriptionLatest(this.props);
```

Comment:
You can avoid a lot of unnecessary calls by only calling `checkSubscriptionLatest` if you are subscribed to that channel

Reply:
good call